eth2deposit/key_handling/key_derivation/tree.py

```diff
@@ -37,8 +37,14 @@ def _parent_SK_to_lamport_PK(*, parent_SK: int, index: int) -> bytes:
     return compressed_PK
 
 
-def _HKDF_mod_r(*, IKM: bytes) -> int:
-    okm = HKDF(salt=b'BLS-SIG-KEYGEN-SALT-', IKM=IKM, L=48)
+def _HKDF_mod_r(*, IKM: bytes, key_info: bytes=b'') -> int:
+    L = 48
+    okm = HKDF(
+        salt=b'BLS-SIG-KEYGEN-SALT-',
+        IKM=IKM + b'\x00',
```

Comment:
By the way, do you know what's the original rationale of adding `I2OSP(0, 1)`?

IMHO it's a bit confusing of what variable `IKM` should be, although I understand `IKM` in different in BLS v2 spec and RFC5869. Maybe add some comments for it,
```suggestion
        IKM=IKM + b'\x00',  # add postfix `I2OSP(0, 1)`
```